package/health_daemon/src/utils.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2018 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <libsbp/sbp.h>
+#include <string.h>
+
+#include "utils.h"
+
+static const char * const bool_names[] = {"False", "True"};
```

Comment:
I don't think these are always capitalized like this, consider using `strcasecmp`.